app/pages/lab-fem/components/layout-options.jsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+
+export default function LayoutOptions({ workflow = null }) {
+  const handleNaturalSubjectDimesions = (e) => {
```

Comment:
```suggestion
  const handleNaturalSubjectDimensions = (e) => {
```

Reply:
In [fem-multi-image-subject-options-editor](https://github.com/zooniverse/Panoptes-Front-End/blob/c532227e78f788f6674480779cf8e95a53caf38d/app/pages/lab-fem/components/fem-multi-image-subject-options-editor.jsx#L18-L23) the handler is a regular function, maybe this should be as well for consistency, or if going to keep as arrow function, don't need `return`.

Optional suggestion is to change this to a regular function.